[2025-08-07 20:33:39,379] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_task_group.section_2.inner_section_2.task_2 scheduled__2025-08-06T20:33:06.678061+00:00 [queued]>
[2025-08-07 20:33:39,445] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_task_group.section_2.inner_section_2.task_2 scheduled__2025-08-06T20:33:06.678061+00:00 [queued]>
[2025-08-07 20:33:39,446] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 20:33:39,446] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-08-07 20:33:39,446] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 20:33:39,551] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): section_2.inner_section_2.task_2> on 2025-08-06 20:33:06.678061+00:00
[2025-08-07 20:33:39,564] {standard_task_runner.py:52} INFO - Started process 383 to run task
[2025-08-07 20:33:39,617] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_task_group', 'section_2.inner_section_2.task_2', 'scheduled__2025-08-06T20:33:06.678061+00:00', '--job-id', '22', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_task_group.py', '--cfg-path', '/tmp/tmpy0mcqfu4', '--error-file', '/tmp/tmp8wgz58ce']
[2025-08-07 20:33:39,643] {standard_task_runner.py:80} INFO - Job 22: Subtask section_2.inner_section_2.task_2
[2025-08-07 20:33:40,050] {task_command.py:369} INFO - Running <TaskInstance: example_task_group.section_2.inner_section_2.task_2 scheduled__2025-08-06T20:33:06.678061+00:00 [running]> on host 034e74559af0
[2025-08-07 20:33:45,030] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_task_group
AIRFLOW_CTX_TASK_ID=section_2.inner_section_2.task_2
AIRFLOW_CTX_EXECUTION_DATE=2025-08-06T20:33:06.678061+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-06T20:33:06.678061+00:00
[2025-08-07 20:33:45,058] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-07 20:33:45,063] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2025-08-07 20:33:45,256] {subprocess.py:85} INFO - Output:
[2025-08-07 20:33:45,275] {subprocess.py:92} INFO - 1
[2025-08-07 20:33:45,278] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-07 20:33:45,449] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_task_group, task_id=section_2.inner_section_2.task_2, execution_date=20250806T203306, start_date=20250807T203339, end_date=20250807T203345
[2025-08-07 20:33:45,583] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-07 20:33:45,997] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
